### ISSUE TITLE
Remove !important from core CSS files

### ISSUE THREAD

**Issue by wpacademy** [ultimatemember/ultimatemember#1267]
**Expected behavior**
Remove the use of !important in the core CSS file..

**Describe the bug**
It is annoying while styling the form fields via some custom CSS. The plugin is using !important on almost all form fields that are not the standard at all. Users should be able to easily apply the custom CSS from a theme or in the customizer.

**Expected behavior**
There should not be any !important at all in the core CSS. Or there should be an option in the plugin setting to disable the use of core CSS files. 
I believe this has been an issue for years now. [407](https://github.com/ultimatemember/ultimatemember/issues/407) - [136](https://github.com/ultimatemember/ultimatemember/issues/136)

**Screenshots**
![image](https://github.com/ultimatemember/ultimatemember/assets/38207694/7b10d84e-c9a6-4263-bf43-fcc097e42c04)

**Do you use UM extensions?**
- [ ] Followers
- [ ] ForumWP
- [ ] Friends
- [ ] Google reCAPTCHA
- [ ] Groups
- [ ] Instagram
- [ ] MailChimp
- [ ] myCRED
- [ ] Notices
- [ ] Online Users
- [ ] Private Content
- [ ] Private Messages
- [ ] Profile Completeness
- [ ] Real-time Notifications
- [ ] Social Activity
- [ ] Social Login
- [ ] Terms & Conditions
- [ ] Unsplash
- [ ] User Bookmarks
- [ ] User Photos
- [ ] User Reviews
- [ ] User Tags
- [ ] Verified Users
- [ ] WooCommerce

**Comment by gwmbox** [comment]
Agree, this has been an issue for many years as you have said.  Core files should not try and over-ride my themes CSS.  Why is this still an issue?

**Comment by MissVeronica** [comment]
You can try this new "UM Comment CSS important" plugin, which will comment over 1300 !important cases.

https://github.com/MissVeronica/um-comment-css-important
